doc/build/kconfig/setting.rst

```diff
@@ -92,7 +92,7 @@ this:
       # CONFIG_SOME_OTHER_BOOL is not set
 
    This is the format you will see in the merged configuration in
-   :file:`zephyr/.config`.
+   :file:`build/zephyr/.config`.
```

Comment:
User can specify the build directory, so the path you have here is not always correct

Reply:
I hadn't considered that. I'll reword accordingly.
Thanks!